admin_guide/configuring_authentication.adoc

```diff
@@ -484,7 +484,7 @@ the correct client certificate and header:
 ----
 # curl -L -k -H "X-Remote-User: joe" \
   --cert /etc/pki/tls/certs/authproxy.pem
```

Comment:
@abutcher Just noticed this is missing a backslash, too. While you're in here, would you mind tacking that on? :ok_woman: